packages/stryker-javascript-mutator/src/mutators/StringLiteralMutator.ts

```diff
@@ -22,7 +22,8 @@ export default class StringLiteralMutator implements NodeMutator {
       }
 
       nodes.push(mutatedNode);
-    } else if (types.isStringLiteral(node) && !types.isImportDeclaration(node.parent) && !types.isJSXAttribute(node.parent)) {
+    } else if ((node.parent === undefined || (node.parent && !types.isImportDeclaration(node.parent) && !types.isJSXAttribute(node.parent)))
```

Comment:
Can be simplified:

`if (!node.parent || (!types.isImportDeclaration(node.parent) && !types.isJSXAttribute(node.parent)))`